Header header
geometry_msgs/Point robot_pos
geometry_msgs/Point goal
float64 dist_from_goal
geometry_msgs/Point collision
uint8 iteration
